### DISCUSSION TITLE
New PICO-8 Emulator?

### DISCUSSION THREAD

**Discussion by beckkramer** [eduardofilo/RG350_adam_image#288] [Ideas & Requests]
Hello! Since [Tac08 has not been updated since October 2020](https://0xcafed00d.itch.io/tac08-rg350), and since it can struggle with newer PICO-8 carts, I was wondering if there were plans to switch to or add another emulator? I was able to get [Fake-08](https://github.com/jtothebell/fake-08) up and running pretty quickly with some manual edits, and so far it's been able to handle all carts I've tried on my 280V. Would love to hear if there are plans in the works on this front, or if opening a PR with the changes I've made would be a useful next step if this sounds interesting to you. Thanks for all your hard work on this project!

**Comment by eduardofilo** [comment]
Of course I am interested in what you tell me. I am a bit distant from the community (too many open fronts), so I thank you for the suggestion. I have installed this emulator and indeed it seems to work very well. I will install it in the next version and I will surely remove tac08. Thank you very much.

**Reply by eduardofilo** [reply]
Done in d299163a445cf7b387f581e5f2422fff0a4cac22 and b23d2bffce9820597ac4b2a7b594670dca8bea08.

**Reply by beckkramer** [reply]
Oh wonderful! Thank you for this and again for all your hard work on this project, it's been great to use.

**Reply by Sneakers8b** [reply]
So I am not sure how to get this emulator working on my 280v I added the renamed fake 8_libretro.opk to the external card and then moved it to the internal card under the core files in retro arch, and then renamed it .so. but it still fails to run the core. am I missing a step?

**Reply by a-dekker** [reply]
It is not a retroarch core, but a standalone emulator.
See also commit https://github.com/eduardofilo/RG350_adam_image/commit/b23d2bffce9820597ac4b2a7b594670dca8bea08 (the green part)
